devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -0,0 +1,15 @@
+# Node.js with React
+# Build a Node.js project that uses React.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- main
```

Comment:
also build dev, no?

Reply:
also [this](https://github.com/1058-Sophia-Gavrila-Forge/Forge-Backend/pull/2/files#r671557965)